### DISCUSSION TITLE
OWASP Machine Learning Security Top 10 - Draft release v0.2 - ML01:2023 Input Manipulation Attack

### DISCUSSION THREAD

**Discussion by giscus** [OWASP/www-project-machine-learning-security-top-10#115] [Machine Learning Security Topics]
# OWASP Machine Learning Security Top 10 - Draft release v0.2 - ML01:2023 Input Manipulation Attack

https://mltop10.info/ML01_2023-Input_Manipulation_Attack.html

**Comment by ankitloud** [comment]
Attack vector Exploitability: 5 (Easy) doesn't look so easy. Since attack requires technical knowledge of deep learning and image processing techniques. How someone first get into the data source and get the data and then exploit. In the same way impact should be high rather difficult since if data is manipulated then it can generate lot of unexpected results and few of them could be life threating as well.
